core/src/main/java/com/klaytn/caver/methods/response/AccessListResult.java

```diff
@@ -22,7 +22,7 @@
 import com.fasterxml.jackson.databind.JsonNode;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
-import com.klaytn.caver.transaction.type.AccessTuple;
+import com.klaytn.caver.transaction.type.transactionUtils.AccessTuple;
```

Comment:
Can you change to `com.klaytn.caver.transaction.transactionUtils.AccessTuple;`?
Or we can change to `com.klaytn.caver.transaction.type.utils.AccessTuple;`.
Which one do you prefer? And if there is something more reasonalbe name, please let me know; 
Actually i don't want to put utils here..